magnusrodseth/systemsoft
# Fetch data using GraphQL and URQL state management

Closes: #5

**Note**

Creating queries for all entities should not be necessary yet. I think you should create queries **after** we have filled up the backend with some test data.

**Changelog**

- Create Employees GraphQL endpoint
- Fetch data using URQL state management

**How to test**

- Spin up application
- Navigate to ex. `/employees`
- Observe that data from Strapi backend gets fetched
- Observe that an error message is display if no data could be fetched
- Observe that a loading UI skeleton is displayed where the data should appear
